rootfs/etc/profile.d/inputrc.sh

```diff
@@ -0,0 +1,4 @@
+# If we do not have one of our own, source inputrc from localhost.
+[ ! -f ~/.inputrc ] && [ -f /localhost/.inputrc ] &&
```

Comment:
Prefer to use `if [ ... ]; then` syntax than long chaining commands.

Reply:
Updated.